test/db/tools/rz_find

```diff
@@ -260,3 +260,17 @@ EXPECT=<<EOF
 EOF
 RUN
 
+NAME=rz-find -E with echo as a command
+FILE==
+CMDS=!rz-find -E "echo" bins/arm/README
```

Comment:
Change the test so that it woudl search for a string `README`, like in previous tests, like the recursive test, on a directory. It should also only print the filename with the `echo`, I think.

Reply:
Suree